src/plugins/transit-demand/TransitLayers.tsx

```diff
@@ -330,9 +336,9 @@ export default function Component({
         parameters: { depthTest: false },
       })
     )
-  
+
   // PIE CHARTS
-  // if (slices.length) {
+  if (slices) {
```

Comment:
slices is an array, not an object. This if is meaningless (always true) because slices is set to {} at the beginning or [] from the useMemo() function. Change it back to `if (slices.length)`

Am I missing something? Is there a reason you changes slices to {} ?